src/modules/filelist/AddFolder.jsx

```diff
@@ -20,15 +21,18 @@ const AddFolder = ({
   onAbort,
   extraColumns
 }) => {
+  const { t } = useI18n()
+  const { showAlert } = useAlert()
+
   if (!visible) {
     return null
   }
 
   return (
     <AddFolderRow
       isEncrypted={isEncrypted}
-      onSubmit={onSubmit}
-      onAbort={onAbort}
+      onSubmit={name => onSubmit(name, showAlert, t)}
+      onAbort={onAbort(showAlert, t)}
```

Comment:
it shouldn't be put in an anomy function to prevent it from being executed when the component is rendered

Reply:
corrigé